anvio/biochemistry/reactionnetwork.py

```diff
@@ -1080,7 +1080,7 @@ class KODatabase:
     Unless an alternative directory is provided, the database is downloaded and set up in a
     default anvi'o data directory, and loaded from this directory in network construction.
     """
-    default_dir = os.path.join(os.path.dirname(ANVIO_PATH), 'data/MISC/KEGG/KO_REACTION_NETWORK')
+    default_dir = os.path.join(os.path.dirname(ANVIO_PATH), 'data/misc/KEGG/KO_REACTION_NETWORK')
```

Comment:
I corrected this but this seems to be going back to what it was, @semiller10

Reply:
I looked at it again, and `misc` should be lowercase. The error was not there for KEGG, but for ModelSEED, where I had it uppercase.

Reply:
@meren See this issue https://github.com/merenlab/anvio/issues/2142#issuecomment-1761684115

Reply:
So all 'misc' in that file are lowercase now? Sorry I'm very confused :)

Reply:
So was I :) Turns out it's lowercase throughout the codebase.